data/gegno/gegno intro missions.txt

```diff
@@ -439,6 +439,168 @@ mission "Giaru Gegno: Revisit"
+				decline
+
+			label "follow"
+			`	You inch your ship forward in acknowledgment. All of a sudden, the second warship on the ground lifts off and forms up behind you, while one in front of you turns around then begins to fly off into the distance. You follow them, curious of where they might be leading you. To your rear, the remaining shuttle takes off, but instead of following you, it heads off into the atmosphere above.`
+			`	Although this planet initially appeared dull due to the several layers of thick clouds, the surface shows occasional signs of life such as plains of grass or occasional dried-up riverbeds, while patches of low fog cover various other locations. As the flight continues, the two Scin ships slowly start to move around so that one is on either side of you. Up ahead in the distance, a large structure comes into view from behind another wall of fog. After getting a bit closer to it, you make out the familiar green colors of the ships flying with you now. The two warships begin to descend towards what you assume to be a sizeable Scin freighter sitting in a large open area of sand.`
+			`	Your previous encounter with a Vi Warror was chaotic and unpredictable, so it's best to be extra cautious in this situation. As you set your ship down near the vessel, the two warships that guided you here also land nearby and power down. Other than the Gegno ships in front of you, there doesn't seem to be any strange activity on your radar at the moment. Environmental readings of the outside surprisingly indicate conditions safe for a human, but it's better to be safe than sorry. You equip your environment suit and your laser pistol just in case.`
+			choice
+				`	(Approach the Scin.)`
+					goto "approach"
+				`	(Stay by your ship.)`
+			`	You take one or two steps forward but remain close by your ship, staying alert. The freigher is indeed large - maybe four times the size of a Hauler, or almost two times the size of a Behemoth. An elevator lowers from underneath, and a group of Scin exit the ship, making their way towards you, their attire being much different than that of the Vi. Most of them are wearing what looks like segmented armor with many layers of plates, giving off the appearance of scales. One of them, however, has a much lighter armor set focused mainly around their shoulders, with orange robe-like garments underneath, and are slightly thinner than the other Scin. Although none of them are nowhere near as bulky as the Vi, they all still appear slightly taller than a human. More importantly, they seem far more eager to interact with you, seeing that they left the comfort of their own ship to greet you.`
```

Comment:
```suggestion
			`	You take one or two steps forward but remain close by your ship, staying alert. The freigher is indeed large - maybe four times the size of a Hauler, or almost two times the size of a Behemoth. An elevator lowers from underneath, and a group of Scin exit the ship, making their way towards you, their attire being much different than that of the Vi. Most of them are wearing what looks like segmented armor with many layers of plates, giving off the appearance of scales. One of them, however, has a much lighter armor set focused mainly around their shoulders, with orange robe-like garments underneath, and are slightly thinner than the other Scin. Although none of them are anywhere near as bulky as the Vi, they all still appear slightly taller than a human. More importantly, they seem far more eager to interact with you, seeing that they left the comfort of their own ship to greet you.`
```
Double negative.
Alternatively, you could say "Although they are nowhere near as bulky as the Vi,"